prdoc/pr_5995.prdoc

```diff
@@ -0,0 +1,19 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Use frame umbrella crate in pallet-proxy and pallet-multisig
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Extends the umbrella crate and uses it in pallet-proxy and pallet-multisig.
+      Migrates benchmarking from v1 to v2 for pallet-proxy and pallet-multisig.
+      Allows CI to pick the umbrella crate weights template to run benchmarks.
+
+crates:
+  - name: pallet-multisig
+    bump: minor
```

Comment:
This seems correct to me -- the interface of these two pallets is the same, but altering deps is a backwards compatible new feature, and should be minor, not patch. 

Can you double check this please @ggwpez?

Reply:
Yea, in these simple cases the CI should also fail if its incorrectly attributed a patch. So just try patch.